packages/frontend/editor-ui/src/views/Evaluations.ee/TestRunDetailView.vue

```diff
@@ -9,13 +9,45 @@ import type { BaseTextKey } from '@n8n/i18n';
 import { useEvaluationStore } from '@/stores/evaluation.store.ee';
 import { useWorkflowsStore } from '@/stores/workflows.store';
 import { convertToDisplayDate } from '@/utils/formatters/dateFormatter';
-import { N8nText, N8nTooltip, N8nIcon } from '@n8n/design-system';
+import {
+	N8nText,
+	N8nTooltip,
+	N8nIcon,
+	N8nTableHeaderControlsButton,
+	N8nExternalLink,
+} from '@n8n/design-system';
 import { computed, onMounted, ref } from 'vue';
 import { useRouter } from 'vue-router';
 import orderBy from 'lodash/orderBy';
 import { statusDictionary } from '@/components/Evaluations.ee/shared/statusDictionary';
 import { getErrorBaseKey } from '@/components/Evaluations.ee/shared/errorCodes';
-import { getTestCasesColumns, mapToNumericColumns } from './utils';
+import {
+	applyCachedSortOrder,
+	applyCachedVisibility,
+	getDefaultOrderedColumns,
+	getTestCasesColumns,
+	getTestTableHeaders,
+} from './utils';
+import { indexedDbCache } from '@/plugins/cache';
+import { jsonParse } from 'n8n-workflow';
+
+export type Column =
+	| {
+			key: string;
+			label: string;
+			visible: boolean;
+			numeric?: boolean;
+			disabled: false;
+			columnType: 'inputs' | 'outputs' | 'metrics';
+	  }
+	| { key: string; disabled: true };
```

Comment:
Could you please add a comment here, why do we exclude some props for the disabled state, and also what's the meaning of `disabled` here and how is it different from `visible`?